test-end-to-end/android-maestro/e2e-tests/003_003_add_existing_content_in_library_as_block.yaml

```diff
@@ -0,0 +1,68 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/003_003_add_existing_content_in_library_as_block"
+  TESTNAME: "003_003_add_existing_content_in_library_as_block"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+  #---------add content to library ---------#
+- runFlow:
+    file: "../common/add_content.yaml"
+    env:
+      TESTFILENAME: "H5p_Content.h5p"
+      TESTCONTENTNAME: H5PContent
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+#---------add content to library ---------#
+- runFlow:
+    file: "../common/add_content.yaml"
+    env:
+      TESTFILENAME: Epub_Content.epub
+      TESTCONTENTNAME: EpubContent
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+#---------add course ---------#
+- runFlow:
+    file: "../common/add_course.yaml"
+    env:
+      TESTCOURSENAME: "Existing Content Block Test"
+#----add teacher to the course----#
+- runFlow:
+    file: "../common/add_new_teacher_inside_course.yaml"
+    env:
+      TESTCOURSENAME: "Existing Content Block Test"
+      TEACHERFIRSTNAME: "Anupama"
+      TEACHERLASTNAME: "Sebastian"
+      GENDER: "Female"
+      PASSWORD: "test"
+#--- add epub content already added by admin in library to course block----#
+- back
+- back
+- tapOn: "OVERVIEW"
+- runFlow:
+    file: "../common/add_existing_content_block_to_course.yaml"
+- tapOn: "EpubContent"
+- tapOn: "SAVE"
+- tapOn: "SAVE"
+- runFlow: "../common/clear_state_and_get_started.yaml"
```

Comment:
Why do we need to clear state here? I think we could add an epub and h5p to the same course. That would make the test faster.

Reply:
In this test admin user add 2 content to the library and then add a new course and add a teacher to that course
This test tests that :
1) Is the admin user able to add content to the course
2) Is the teacher user able to add content to the course

So we clear state here for teacher login

Reply:
Admin adds epub content and teacher adds h5p content in this test